js/server/modules/@arangodb/foxx/manager.js

```diff
@@ -660,57 +642,48 @@ function _deleteServiceFromPath (mount, options) {
 // FoxxService.basePath(mount) and FoxxService.bundlePath(mount). This is to
 // avoid a race condition with selfHeal() which could delete the files before
 // the service is saved to the database.
-function _install (mount, tempPaths, options = {}) {
+function _install (tempService, tempBundlePath, options = {}) {
```

Comment:
We need to create the temporary service to validate it anyway, so it's easier to just pass it through rather than recreate it every time.